### ISSUE TITLE
show file and line number on global logger

### ISSUE THREAD

**Issue by thewh1teagle** [gabime/spdlog#3257]
Hi!

I would like to show filename and line number in the global logger.
I tried to use the macros like

```console
SPDLOG_DEBUG(...)
```
but seems like it depends on constant static level defined with `#define` otherwise it's not print anything (also I can't change it?). However I would like to simply use:

```console
spdlog::debug(...)
```
And it will show line numbers.

If it's not possible and it's possible only when specifying the logger, should I declare the logger statically in some header file and use it directly eg.

```console
spdlog::debug(g_logger, ...)
```

TLDR of what I'm looking for:

```console
int main() {
  spdlog::cfg::load_env_levels();
  spdlog::debug("hello");
  return 0;
}
./app
DEBUG main.cpp:2: hello
```

The project for repo: [loud.cpp](https://github.com/thewh1teagle/loud.cpp)

**Comment by tt4g** [comment]
The logging macro was originally introduced as a feature to remove unnecessary log output at compile time: https://github.com/gabime/spdlog/wiki/0.-FAQ#how-to-remove-all-debug-statements-at-compile-time-

Replace all logging statements with logging macros and define `#define SPDLOG_ACTIVE_LEVEL SPDLOG_LEVEL_TRACE` macro (before using spdlog) so that logging macros are not removed.
